app/src/README.md

```diff
@@ -1,17 +1,29 @@
-Node/Grunt modules for backend’s frontend workflow.
+# Bolts backend’s frontend workflow using grunt
```

Comment:
Pluralisation and possession is out slightly. Probably better with "Bolt's backend/frontend"